Client/Application/NavigationRouter.swift

```diff
@@ -111,10 +111,6 @@ enum NavigationPath {
             if (urlString.starts(with: "http:") ||  urlString.starts(with: "https:") && UserDefaults.standard.bool(forKey: "OpenedAsDefaultBrowser")) {
                 TelemetryWrapper.gleanRecordEvent(category: .action, method: .open, object: .asDefaultBrowser)
                 UserDefaults.standard.set(true, forKey: "OpenedAsDefaultBrowser")
-                /*
-                 Ecosia
-                 */
-                Analytics.shared.defaultBrowser()
```

Comment:
I'm just a little bit confused as Mozilla themselves track their default browser events in here. There is even a comment that explains it. But maybe is misleading?

Reply:
Yes true, but it seemed to never have worked - I checked the current implementation (release/32.0) and there it's at the place where I put it